dynawo/3rdParty/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Choose the type of build, options a
 option(CXX11_ENABLED "Enable support of C++11 standard" TRUE)
 set(MSVC_STATIC_RUNTIME_LIBRARY OFF CACHE BOOL "Choose either ON or OFF.")
 set(BOOST_ROOT_DEFAULT "true" CACHE STRING "Enable to find boost in system path or not.")
+set(FORCE_CXX11_ABI OFF CACHE BOOL "Choose either ON or OFF.")
```

Comment:
Could you please elaborate on why we need this? 
Doesn't compilation with latest versions of gcc cover this?

Reply:
To be the most compatible we need to compile with the oldest gcc version possible with real c++11 support, it is 5.1, I did not found a Docker image with gcc 5.1 but fedora 22 has 5.3.1 which is close enough. This gcc is compiled with the option `--with-default-libstdcxx-abi=gcc4-compatible` which does not activate the new abi introduced in 5.1. And we need this new abi activated to compile on newer gcc.
https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html

Reply:
We shall be careful that being able to compile on old versions of cxx11 might prevent us from using modern coding styles. The issue is not true today as anyway we are limited by cpp98 but this might be an issue tomorrow. I propose that we merge this, but we keep in mind that later we might drop this compatibility

Reply:
gcc 5.1 implements the full c++11 standard and there is no issue if we use c++11 in newer gcc. But it would not be the same if we want to use c++14 feature...

Reply:
yes, but as I mentioned it is not for next week ;)